lib/vanity/experiment/ab_test.rb

```diff
@@ -445,6 +445,14 @@ def destroy
         connection.destroy_experiment @id
         super
       end
+      
+      # clears all collected data for the experiment
+      def reset
```

Comment:
Makes sense to define AbTest#reset instead of defining it on each adapter - I like it. Do we need to re-initialize the experiment here, or would that happen on the next page load that uses the experiment?